README.md

```diff
@@ -231,6 +232,9 @@ new feature and has not been well tested.  To do this:
     unzip api.zip
     cd ./IBJts/source/pythonclient
     python3 setup.py install
+    # cleanup leftover files
+    cd ../../..
+    rm -rf api.zip IBJts META-INF
```

Comment:
I don't think it is necessary to document the cleanup.  Anyone familiar with Linux can do this if they want.  They may actually want the IBJts around for some other reason.